spec/factories/miq_alert_status_label.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :miq_alert_status_label do
+    name 'mylabelname'
+    value 'mylabelvalue'
+
+    # By default `FactoryGirl` calls the `save!` method when the instance is created, but we don't
+    # want to do that for this class because it isn't persisted to any table, and it doesn't have
+    # that `save!` method.
```

Comment:
call `.build` on your objects instead?

Reply:
Yes please.
https://robots.thoughtbot.com/tips-for-using-factory-girl-without-an-orm
This blog post suggests `skip_create` as technical workaround to pass `.lint` though I don't think we do factory linting in manageiq.  Even if you declare this, you can call `.build` in test, I really think that will be less confusing to read.